data/projects.json

```
@@ -58,5 +58,9 @@
   "gatsby": {
     "name": "Gatsby",
     "q": "org:gatsbyjs is:issue is:open label:\"good first issue\""
+  },
+  "babel": {
+    "name": "Babel",
+    "q": "repo:babel/babel is:issue is:open label:\"good first issue\""
```

Thank you for the PR!

Looking at Babel's list of Good First Issues, it looks like they also have Labels for `claimed` and `Has PR`. I think it would be helpful if we filtered those labels out so we get issues that are truly unclaimed Good First Issues that someone could get started with right away.

Here's one way you could do that by adding just a bit to the search:

```suggestion
    "q": "repo:babel/babel is:issue is:open label:\"good first issue\" -label:\"Has PR\" -label:\"claimed\""
```

Because of this PR, I [learned that](https://help.github.com/articles/understanding-the-search-syntax/) `-` in front of any query or filter effectively acts as `NOT` meaning that searches matching those results will not be included!

---

Good to know! Thanks for pointing that out, adding this now